packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -296,6 +296,20 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
 
         return configFile;
       },
+      '0.25.12': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([, options]) => {
+            if (options.network === NETWORK_TESTNET) {
+              options.core.docker.image = base.get('core.docker.image');
+
+              options.platform.drive.tenderdash.genesis.chain_id = testnet.get('platform.drive.tenderdash.genesis.chain_id');
```

Comment:
base configuration also has network testnet. We need to make sure it's not a base